JSON_Schemas/Weights_and_Condition_Scores/iscScoreEventCore.json

```diff
@@ -0,0 +1,39 @@
+{
+    "required": [
+    ],
+
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "icarEventCoreResource.json"
+        },
+        {
+            "properties": {
+                "assessor": {
+                    "type": "string",
+                    "description": "Name of the assessor"
+                },
+                "iscFatScore": {
+                    "type": "object",
+                    "$ref": "iscFatScore",
+                    "description": "Event object for capturing Fat Score details"
+                },
+                "iscConditionScore": {
+                    "type": "object",
+                    "$ref": "iscConditionScore",
+                    "description": "Event object for capturing Condition Score details"
+                },
+                "iscFrameScore": {
+                    "type": "object",
+                    "$ref": "iscFrameScore",
+                    "description": "Event object for capturing Frame Score details"
+                },
+                "iscMuscleScore": {
+                    "type": "object",
+                    "$ref": "iscMuscleScore",
+                    "description": "Event object for capturing Muscle Score details"
+                }
```

Comment:
I thought the score core event was to provide for assessor, and there would be individual score events for each of the types, rather than building them into iscScoreEventCore??

Reply:
Change is done in the current version of the schema